package.json

```diff
@@ -3,18 +3,22 @@
   "version": "0.0.0",
   "description": "JS lib for interacting with Textile APIs.",
   "scripts": {
+    "docs:markdown": "api-documenter markdown --input-folder=temp",
+    "docs:html": "npm run docs:markdown && cp -r markdown/* docs/ && cp packages/hub/README.md ./docs/",
+    "build:docs": "npm run docs:markdown && npm run docs:html",
+    "build:extractor": "api-extractor run --local --verbose",
+    "build:lerna": "lerna run build",
     "bootstrap": "lerna bootstrap",
     "compile": "lerna run compile",
-    "build": "lerna run build",
+    "build": "npm run build:lerna && npm run build:extractor && npm run build:docs",
```

Comment:
okay, so docs will regenerate anytime it's built now. 

the docs folder is checked into the repo. so we should be able to just change our docs deployment choice and it'll be live.

Reply:
hmmm, this is a bummer (having the docs folder checked into the repo), is this just for the 'static' assets? Not the generated docs right?